docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -212,14 +245,34 @@ properties must be defined.
 The type of kNN algorithm to use. Currently only `hnsw` is supported.
 
 `m`:::
-(Required, integer)
+(Optional, integer)
 The number of neighbors each node will be connected to in the HNSW graph.
 Defaults to `16`.
 
 `ef_construction`:::
-(Required, integer)
+(Optional, integer)
 The number of candidates to track while assembling the list of nearest
 neighbors for each new node. Defaults to `100`.
+
+`quantization_options`:::
+(Optional, object)
+An optional section that configures the quantization configuration. The
+quantization configuration is used to reduce the memory footprint of the
+index. Only `byte` quantization is currently supported and can only be configured if
+`element_type` is `float`.
+.Properties of `quantization_options`
+[%collapsible%open]
+=====
+
+`type`:::
+(Required, string)
+The type of quantization. Currently only `byte` is supported.
+
+`confidence_interval`:::
+(Optional, float)
+The confidence interval to use when quantizing the vectors, can be any value between and including `0.90` and `1.0`.
+Defaults to `1/(dims + 1)`.
```

Comment:
Maybe explain what the confidence interval does? And also why not using `configured_quantile` like in the format ctr?

Reply:
@jimczi I think we should rename it in the ctor. Its really the confidence interval. I will rename it in Lucene

Reply:
Let me get @tveasey 's opinion. What do you think. `confidence_interval` or `quantile`?

Reply:
As we discuss off line:

The q'th quantile x is a single point in the distribution curve, i.e. a value that x such that F(x) = q. A (central) p'th confidence interval is a pair of values (x, y) such that F(x) = 1 - p/2 and F(y) = 1 + p/2

So I believe strictly this is working in terms of a confidence interval since we find a pair or values we use to clamp values.